elixir-phoenix-app-deployment.md

```diff
@@ -0,0 +1,339 @@
+
+Once you have logged into your Heroku account
+and are viewing your "dashboard": https://dashboard.heroku.com
+
+1.1 Click on the `New` button, then <br />
+1.2 Click on `Create new app`:
+
+![heroku-create-app](https://user-images.githubusercontent.com/194400/36428249-575cf816-1647-11e8-9005-0ea5219a039c.png)
+
+### 2. Give Your New App a _Name_
+
+Input your desired name
+(_if it's available; otherwise get **creative**!_): <br />
+![heroku-new-app-name-defaults](https://user-images.githubusercontent.com/194400/36433381-44f30258-1654-11e8-8af8-a3e262a28573.png) </br >
+In this case we are creating an App called `phxchat`.
+(_short for `Phoenix Chat`,
+  because **obviously** `phoenixchat` is "taken"..._)
+
+Leave the rest of the options to the "default" options
+(_unless you have specific needs_).
+
+> _**Note**: don't worry about the "Add to pipeline" option for now,
+we will come back to it later_.
+
+Click on the `Create app` button.
+
+You will be directed to the "Deploy" tab for your app: <br />
+![heroku-phx-chat-dwploy-tab](https://user-images.githubusercontent.com/194400/36435297-31653eb8-1659-11e8-9ce7-2c47ad38471c.png)
+
+### 3. Create "Deployment Pipeline"
+
+On the "Deploy" tab of your app, scroll down
+till you see the "Deployment Method" section:
+
+![heroku-deploy-connect-to-github-connect](https://user-images.githubusercontent.com/194400/36437399-c76b9664-165e-11e8-99a6-4e72756705ae.png)
+
+1. Click on "GitHub"
+2. Select the "Owner" of the App you want to deploy
+(_usually your own GitHub username_)
+3. Type (_or copy-paste_) the name of your App.
+4. Click `Search` to find the repo.
+5. Click "Connect" button.
+
+You should see: <br />
+![heroku-app-connected](https://user-images.githubusercontent.com/194400/36438050-97d7c132-1660-11e8-979b-0846fcacc9bc.png)
+
+
+### 4. Add the PostgreSQL Database "Add-on"
+
+Given that our Phoenix App uses a Postgres database to store it's data,
+let's add it as an "Add-on".
+
+Scroll to the top of the dashboard and click on the "Resources" tab:
+
+![click-on-resources-tab](https://user-images.githubusercontent.com/194400/36438338-83dc0b06-1661-11e8-89a0-3268638a4c34.png)
+
+On the "Resources" page, scroll down till you See "Add-ons"
+
+![heroku-addons-select-postgres](https://user-images.githubusercontent.com/194400/36438438-d6bcd350-1661-11e8-89fe-b0b5889c8f93.png)
+
+1. Type: "post" (_in the add-on search input box_)
+2. Select "Heroku Postgres" from the list.
+
+This will open a "Modal" for you to _confirm_: <br />
+![heroku-postgres-select-default](https://user-images.githubusercontent.com/194400/36438710-9ae7b2ea-1662-11e8-829b-c1ee854b54d5.png)
+
+Leave the _default_ "Hobby Dev - Free"
+and _click_ the "**Provision**" button.
+
+You should now see (_something similar to_) the following: <br />
+![heroku-addon-installed](https://user-images.githubusercontent.com/194400/36438893-21945f00-1663-11e8-87a4-62baa1f56e95.png)
+
+
+### 5. Create a `elixir_buildpack.config` File
```

Comment:
I think I would put this section just before (or after) the section 9 where the user add the buildpacks manually to Heroku.
I would also add more explanation on what are buildpacks